docs/Actions.md

```diff
@@ -592,6 +592,41 @@ set_changelog(app_identifier: "com.krausefx.app", version: "1.0", changelog: "Al
 
 You can store the changelog in `./fastlane/changelog.txt` and it will automatically get loaded from there. This integration is useful if you support e.g. 10 languages and want to use the same "What's new"-text for all languages.
 
+### upload_artifact
```

Comment:
Sorry, but we'll have to rename this action: artifacts in the context of `fastlane` are the build artifacts, which get usually cleared after the run.

Why not name the action `artifactory`?